### PR TITLE
Add Signup Page With Mailchimp

### LINKED ISSUE #193: Embed the Mailing List Form
I thought the pop up Mailchimp form for the "Join Us" tab is less convenient. Maybe embed the Mailchimp would make more sense? I can work on this if possible.
image

### PR DESCRIPTION
**Pull request by yuchenliu15**
Tested and working correctly
image

### CODE REVIEW

**Review by Kn99HN — COMMENTED** [review]
Thanks for tackling this! Can we also collect first and last name in the form?

**Comment by yuchenliu15** [comment]
@Kn99HN I removed those because I feel people will be too lazy. Wdyt?

**Comment by Kn99HN** [comment]
> @Kn99HN I removed those because I feel people will be too lazy. Wdyt?

I think the form with only email address feels a bit empty in the page. I think we should still collect names. We can combine into 1 input for `Name`

Maybe, instead, we can replace `Join Us` with a mailchimp logo and have a pop up to collect the email address?

@esilverm Thoughts?

**Comment by esilverm** [comment]
I am fine keeping the join us instead of the mailchimp logo. I dont know if the logo will
be intuitive enough for the users. I do think we should include the inputs for names. I will take a look into this stuff a bit more when i get home

**Comment by yuchenliu15** [comment]
@Kn99HN prettier worked! ty!!

**Review by Kn99HN — APPROVED** [review]
LGTM!. Gonna wait for @esilverm  for final approval

File: src/pages/signup.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            />
+          </div>
+          <div className="mc-field-group">
+            <label htmlFor="mce-FNAME">First Name </label>
+            <input type="text" name="FNAME" className="" id="mce-FNAME" />
+          </div>
+          <div className="mc-field-group">
+            <label htmlFor="mce-LNAME">Last Name </label>
+            <input type="text" name="LNAME" className="" id="mce-LNAME" />
+          </div>
```
Comment (esilverm) [inline src/pages/signup.js:47]: Should we put first name and last name inputs over email address? It may look better

File: src/pages/signup.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+
+const FormComtainer = styled.div`
+  width: 30%;
+  margin: auto;
+  #mc_embed_signup {
+    background-color: rgba(255, 255, 255, 0.4);
+    backdrop-filter: blur(5px);
+    padding: 2em;
+    clear: left;
+    font: 20px Helvetica, rial, sans-serif;
```
Comment (esilverm) [inline src/pages/signup.js]:
```suggestion
    font: 20px Helvetica, Arial, sans-serif;
```

**Review by esilverm — COMMENTED** [review]
Just two small notes, otherwise it looks good

File: src/pages/signup.js (hunk 1 above)
Comment (yuchenliu15) [reply src/pages/signup.js:47]: @esilverm then make all fields required? Otherwise since only email is required we might want to put it first

Comment (esilverm) [reply src/pages/signup.js:47]: I guess so, let's just merge and figure this out later

**Review by esilverm — APPROVED** [review]

**Pull request merged**